x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details_content.tsx

```diff
@@ -70,83 +74,98 @@ export const EndpointDetailsContent = memo(
+                defaultMessage="OS"
+              />
+            </ColumnTitle>
+          ),
+          description: <EuiText size="xs">{details.host.os.full}</EuiText>,
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.agentStatus', {
-            defaultMessage: 'Agent Status',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.agentStatus"
+                defaultMessage="Agent Status"
+              />
+            </ColumnTitle>
+          ),
           description: <EndpointAgentStatus hostStatus={hostStatus} endpointMetadata={details} />,
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.lastSeen', {
-            defaultMessage: 'Last Seen',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.lastSeen"
+                defaultMessage="Last Seen"
+              />
+            </ColumnTitle>
+          ),
           description: (
-            <EuiText>
+            <EuiText size="xs">
               {' '}
-              <FormattedDateAndTime date={new Date(details['@timestamp'])} />
+              <FormattedDate value={details['@timestamp']} fieldName="" />
             </EuiText>
           ),
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.policy', {
-            defaultMessage: 'Policy',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.policy"
+                defaultMessage="Policy"
+              />
+            </ColumnTitle>
+          ),
           description: (
-            <EuiFlexGroup alignItems="center">
-              <EuiFlexItem grow={false}>
-                <EuiText>
-                  <EndpointPolicyLink
-                    policyId={details.Endpoint.policy.applied.id}
-                    data-test-subj="policyDetailsValue"
-                  >
-                    {details.Endpoint.policy.applied.name}
-                  </EndpointPolicyLink>
+            <EuiText size="xs" className={'eui-textBreakWord'}>
+              <EndpointPolicyLink
+                policyId={details.Endpoint.policy.applied.id}
+                data-test-subj="policyDetailsValue"
+                style={{ paddingRight: '5px' }}
+              >
+                {details.Endpoint.policy.applied.name}
+              </EndpointPolicyLink>
+              {details.Endpoint.policy.applied.endpoint_policy_version && (
+                <EuiText
+                  color="subdued"
+                  size="xs"
+                  className={'eui-displayInlineBlock'}
+                  style={{ whiteSpace: 'nowrap', paddingRight: '5px' }}
```

Comment:
FYI: I think Eui has a class name that can be added to `className=` prop for no wrap

Reply:
Also we should avoid using the `style` prop and use styled components instead

Reply:
I was wondering what was the go-to approach for this. I'll go for classNane. 

@paul-tavares yes I totally forgot about that, thanks!